server/events/tests/shared_case.py

```diff
@@ -0,0 +1,24 @@
+from datetime import timedelta
+from django.utils import timezone
+from django.test import TestCase
+from django.contrib.auth import models as auth_models
+
+class SharedTestCase(TestCase):
+    """ A base test module that sets up dummy variables for testing. """
+
+    @classmethod
+    def setUpTestData(self):
+        # users
+        self.user1 = auth_models.User.objects.create_user(
+            username = 'user1', email='dragonboi33@user1.com')
+        self.user2 = auth_models.User.objects.create_user(
+            username = 'user2', email='remlover48@user2.com')
+
+        # event1 start/end times
+        self.start_datetime1 = timezone.now()
```

Comment:
As a general rule, simpler values like these should probably be declared closer to where they are used. More complex values (like the models you instantiated above) are pains to re-create, and are re-used all over the place, so it makes sense to share them across tests.

Reply:
fixed